src/react/components/VectorStyle/VectorStyleModal.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { VectorNameInput } from 'oskari-ui/components/VectorStyle';
+import { LocaleConsumer, Controller } from 'oskari-ui/util';
+import { Message, Modal } from 'oskari-ui';
+import { StyleEditor } from 'oskari-ui/components/StyleEditor';
+
+export const VectorStyleModal = LocaleConsumer(({editorState, onCancel, onModalOk, setEditorState, nameValidation, setName}) => {
```

Comment:
Shouldn't need to have LocaleConsumer here since Message tag is a consumer.

Reply:
Fixed.